builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -1052,3 +1051,44 @@ def getBuildModuleUrl() {
     return "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"
   }
 }
+
+def handleDeploymentErrors(deployOutput, serviceName, env, credsId) {
+	if(deployOutput.contains('ROLLBACK_COMPLETE state and can not be updated')){
```

Comment:
same as above comment. Please correct here and other places

Reply:
did the changes